backend/config/urls.py

```diff
@@ -19,19 +19,27 @@
 from django.conf.urls import url
 
 from app.common import render_react_view
-from app.views import africa_map_geojson, democracy_score_json
+from app.views import (
+    africa_map_geojson,
+    budget_response,
+    population,
+    democracy_score_json,
+)
 
 urlpatterns = [
     # Django admin page
     path('admin/', admin.site.urls),
 
     # API endpoints
     path('api/africa_map_geojson/', africa_map_geojson),
+    path('api/budget_response/', budget_response),
+    path('api/population/', population),
     path('api/democracy_scores/', democracy_score_json),
 
     # React views
     url('d3/', render_react_view, {'component_name': 'MapQuizD3'}),
     url('leaflet/', render_react_view, {'component_name': 'MapQuizLeaflet'}),
     url('svg/', render_react_view, {'component_name': 'MapQuizSVG'}),
+    url('simulation/', render_react_view, {'component_name': 'MainView'}),
```

Comment:
better route name, please. `simulation` isn't descriptive -- everything in this project is going to be a simulation